Headers/DebugServer2/Utils/Bits.h

```diff
@@ -100,6 +100,14 @@ template <typename T> static inline void DisableBit(T &number, int idx) {
 }
 
 template <typename T>
+static inline void DisableBits(T &number, int startIdx, int endIdx) {
```

Comment:
If I understand correctly, `startIdx` is inclusive, and `endIdx` is exclusive. Having a comment about this wouldn't hurt.